test/unit/lib/plugins/aws/package/compile/events/alb/lib/listenerRules.test.js

```diff
@@ -61,6 +61,19 @@ describe('#compileListenerRules()', () => {
           },
           authorizers: ['myFirstAuth', 'mySecondAuth'],
         },
+        {
```

Comment:
All newly added tests should be backed by  `runServelress` util (check https://github.com/serverless/serverless/tree/master/test#unit-tests)

This tests file is actually scheduled for refactor to such, and we should not add new tests written this way.

Let's add new test that covers that case here: https://github.com/serverless/serverless/blob/8f450e47e2ef29fad62a44aa25cba70259a31edf/test/unit/lib/plugins/aws/package/compile/events/alb/index.test.js (simply by adding test function that covers test case and corresponding `it` block)